sdk/openai/tools/TestFramework/src/OpenAI.TestFramework.csproj

```diff
@@ -18,7 +18,7 @@
     <PackageReference Include="NUnit3TestAdapter" />
     <PackageReference Include="Moq" />
     <PackageReference Include="System.ValueTuple" />
-    <PackageReference Include="System.ClientModel" VersionOverride="1.1.0-beta.5" />
+    <PackageReference Include="System.ClientModel" />
 
     <ExcludeFromProjectReferenceToConversion Include="System.ClientModel" />
```

Comment:
Can this one go away now too?

Reply:
We will need to add this back every time we update System.ClientModel in this repo, but I will remove it for now since it is no longer needed